compositor_render/src/transformations/shader/validation.rs

```diff
@@ -131,8 +137,8 @@ fn validate_type_equivalent(
         | naga::TypeInner::ValuePointer { .. } => {
             if ti1 != ti2 {
                 return Err(TypeEquivalenceError::TypeStructureMismatch {
-                    expected: type1.inner.clone(),
-                    actual: type2.inner.clone(),
+                    expected: type1.inner.to_string(mod1),
+                    actual: type2.inner.to_string(mod2),
```

Comment:
`type1` and `type2` are pretty bad names. Errors assume that those values have other meanings. Use `expected` and `actual`.

Reply:
I agree, but I don't really want to through all that code